fix: Fixed Typewriter component

zvolcsey opened the pull request:
@rupali-codes @CBID2 Hi! I fixed one of the errors that appeared on the console. `TypeWriter component` returns a `div` and this `div` was inside a `p` tag.

## Fixes Issue

Closes #405 

## Changes proposed

- Removed parent p tag and added classes to the `Typewriter component`.

## Screenshots
![console_error](https://user-images.githubusercontent.com/41576384/230615119-b36626bd-6e6e-446d-a33d-07fec17ae673.png)

CBID2 commented on pages/index.tsx:
```diff
@@ -26,14 +26,16 @@ export default function Home() {
             <span>👾</span>
           </div>
           <div className=" flex flex-col mt-6 justify-center items-start">
```
```suggestion
          <article className=" flex flex-col mt-6 justify-center items-start">
```

CBID2 replied:
```suggestion
          <article className=" flex flex-col mt-6 justify-center items-start">
```

CBID2 commented on pages/index.tsx:
```diff
@@ -26,14 +26,16 @@ export default function Home() {
+                loop: true,
+              }}
+            />
           </div>
```
```suggestion
          </article>
```

CBID2 replied:
```suggestion
          </article>
```

CBID2 replied:
```suggestion
          </article>
```

CBID2 replied:
```suggestion
          </article>
```

CBID2 commented on pages/index.tsx:
```diff
@@ -26,14 +26,16 @@ export default function Home() {
             <span>👾</span>
           </div>
```
```suggestion
          </article>
```

CBID2 replied:
```suggestion
          </article>
```

CBID2 replied:
```suggestion
          </article>
```

CBID2 reviewed (changes requested):
Hi @zvolcsey! I made some suggestions on how to improve your code. It’s best to use `<div>`tags sparingly because they make your code less accessible.

rupali-codes commented:
@CBID2 hey do you have any specific reason why are you replacing all <div> with <article>?

CBID2 commented:
> Hi @zvolcsey! I made some suggestions on how to improve your code. It’s best to use `<div>`tags sparingly because they make your code less accessible. Other than that, good job! :)

I explain why here @rupali-codes

zvolcsey commented:
@rupali-codes @CBID2 Hi! May I ask what the decision is?
Should I change the other `div`-s? or
We keep the original `div`-s.
Only the parent `p` tag needs to be removed.

CBID2 commented:
> @rupali-codes @CBID2 Hi! May I ask what the decision is?
> 
> Should I change the other `div`-s? or
> 
> We keep the original `div`-s.
> 
> Only the parent `p` tag needs to be removed.

Good question @zvolcsey! I personally would use more semantic HTML to replace the `<div>`tags.

rupali-codes commented:
> > @rupali-codes @CBID2 Hi! May I ask what the decision is?
> > Should I change the other `div`-s? or
> > We keep the original `div`-s.
> > Only the parent `p` tag needs to be removed.
> 
> Good question @zvolcsey! I personally would use more semantic HTML to replace the `<div>`tags.

Umm..But then we'll have to replace it from everywhere, @Eugene44-hub do you have any takes on this?

Eugene44-hub commented:
> > > @rupali-codes @CBID2 Hi! May I ask what the decision is?
> > > Should I change the other `div`-s? or
> > > We keep the original `div`-s.
> > > Only the parent `p` tag needs to be removed.
> > 
> > 
> > Good question @zvolcsey! I personally would use more semantic HTML to replace the `<div>`tags.
> 
> Umm..But then we'll have to replace it from everywhere, @Eugene44-hub do you have any takes on this?

So i suggest the article tag should be used like where we have much text. Like the cards that will be ok.

zvolcsey commented:
@rupali-codes @CBID2 @Eugene44-hub Thanks for the feedback! Then I will replace the `article` wtih the original `div`. The issue is soldved without changing the tags and we can improve the structure and the tags later.

zvolcsey commented:
@rupali-codes @CBID2 @Eugene44-hub Hi! In my latest commits, I made the followings improvements:

- Merge the latest commits.
- Replaced the changed `article` tag with the original `div`.

The currently changes:

- Format with Prettier.
- Removed the parent `p` tag and added the styles for the `Typewriter-component`.

I think we can improve the tags in an other issue, if we will find a better solution. The issue is currently solved. The error message disappeared in development mode, because the `p` tag does not have a `div` tag.

CBID2 commented:
> @rupali-codes @CBID2 @Eugene44-hub Hi! In my latest commits, I made the followings improvements:
> 
> 
> 
> - Merge the latest commits.
> 
> - Replaced the changed `article` tag with the original `div`.
> 
> 
> 
> The currently changes:
> 
> 
> 
> - Format with Prettier.
> 
> - Removed the parent `p` tag and added the styles for the `Typewriter-component`.
> 
> 
> 
> I think we can improve the tags in an other issue, if we will find a better solution. The issue is currently solved. The error message disappeared in development mode, because the `p` tag does not have a `div` tag.

Ok that's a good idea

rupali-codes reviewed (commented):
looks good to me, could you please resolve this conversations?

zvolcsey commented:
@rupali-codes Hi! In my latest commit I resolved the merge conflict.

The problem was the `p` tag before the `TypewriterComponent`. 
The `TypewriterComponent` returns a `div`.
In development mode was an error, because the `p` tag had a `div` tag.

rupali-codes commented:
> @rupali-codes Hi! In my latest commit I resolved the merge conflict.
> 
> The problem was the `p` tag before the `TypewriterComponent`. The `TypewriterComponent` returns a `div`. In development mode was an error, because the `p` tag had a `div` tag.

Thanks
